src/plugins/expressions/common/execution/execution.ts

```diff
@@ -489,10 +489,6 @@ export class Execution<
           continue;
         }
 
-        if (!aliases?.length) {
```

Comment:
why this change?

Reply:
Right now, if you'll open the `Canvas` from the main, and try to use the expression, which has the required argument without specified `aliases`, you'll receive `${fnDef.name} requires an argument` message. But it is not informative for the user, because tones of the arguments can be required. On the other hand, this check has been added because of the code above:
```js 
const errorArg = name === '_' ? aliases[0] : name;
```

But `name` can't be `_`, because there is a check for it at the side of the expressions.

https://github.com/elastic/kibana/blob/e8f59a588886f10eb4d37bfd80742a9239190ee9/src/plugins/expressions/common/expression_functions/expression_function_parameter.ts#L30

Possibly, we should revisit the code, mentioned in this comment with comparing the name with `_`, but I believe, it shouldn't be fixed at this PR. WDYT?

Reply:
All good, I just didn't know what this is responsible for and wanted to know 🙆🏼